contracts/0.8.9/oracle/AccountingOracle.sol

```diff
@@ -243,8 +243,8 @@ contract AccountingOracle is BaseOracle {
 
         /// @dev The share/ETH rate with the 10^27 precision (i.e. the price of one stETH share
         /// in ETH where one ETH is denominated as 10^27) used for finalizing withdrawal requests
-        /// up to (and including) the one passed in the lastWithdrawalRequestIdToFinalize field.
-        /// Must be set to zero if lastWithdrawalRequestIdToFinalize is zero.
+        /// up to (and including) the one passed in the lastFinalizableWithdrawalRequestId field.
```

Comment:
this is a subject of change after #651 is merged
though, it's better to have in diff till that